.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "\U0001F680 Feature Request"
+about: Suggest an idea for this project
+labels: feature-request, needs-triage
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Describe the Feature
+<!--- A clear and concise description of the feature you are proposing -->
+
+## Is your Feature Request related to a problem?
+<!--- A description of the issue, e.g. I'm always frustrated when... -->
+
+## Proposed Solution
+<!--- Not required, but suggest how to implement the addition or change -->
+
+## Describe alternatives you've considered
+<!--- Any alternative solutions or features you've considered -->
+
+## Additional Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+
+
+
+
```

Comment:
nit: any reason for all this whitespace?

Reply:
I just wanted to make space and call attention to the checkbox below when the customer is filling the template.